packages/legend-studio/src/components/editor/edit-panel/ModelLoader.tsx

```diff
@@ -187,25 +214,35 @@ export const ModelLoader = observer(() => {
             </button>
           )}
         </div>
-        <div className="model-loader__header__action">
-          <button
-            className="btn--dark model-loader__header__load-btn"
-            onClick={loadModel}
-            disabled={modelLoaderState.isLoadingModel}
-            tabIndex={-1}
-            title="Load model"
-          >
-            Load
-          </button>
-        </div>
+        {!modelLoaderState.currentExtraNativeInputType && (
```

Comment:
Might make sense to add this as part of the extension definition. So we could add a `hardReplaceOption?: boolean | undefined` property on `ExtraModelLoaderRendererConfiguration` to see if we render this or not.